tests/int/conftest.py

```diff
@@ -0,0 +1,32 @@
+'''Common pytest integration code.'''
+import re
+import subprocess
+
+import pytest
+
+
+class Helpers:
+    @staticmethod
+    def run(command, expected_stdout_lines=None, expected_stderr_lines=None,
+            cwd='examples'):
+        '''Helper function to eliminate pytest code.'''
+        if expected_stdout_lines is None:
+            expected_stdout_lines = ['Connected', 'Starting operation', 'Errors: 0']
+        if expected_stderr_lines is None:
+            expected_stderr_lines = ['']
+        results = subprocess.run(command, shell=True, cwd=cwd, capture_output=True)
```

Comment:
The final problem is this line - `subprocess.run` doesn't exist in Py2 :(

You should be able to use popen:

```py
process = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
process.wait()  # wait until we have a returncode
```

Reply:
Ok

At some point, consider deprecating v2.7.